Resources/input/fsh/profiles/RequestBundle.fsh

```diff
@@ -37,10 +38,16 @@ Description: "A communication resource for collecting information objects needed
 * entry[MedicationRequest].response ..0
 * entry[Patient].link ..0
 * entry[Patient].resource 1..
-* entry[Patient].resource only $KBV_PR_FOR_Patient
+* entry[Patient].resource only KBV_PR_FOR_Patient
 * entry[Patient].search ..0
 * entry[Patient].request ..0
 * entry[Patient].response ..0
+* entry[Practitioner].link ..0
+* entry[Practitioner].resource 1..
+* entry[Practitioner].resource only KBV_PR_FOR_Practitioner
```

Comment:
Bitte nach dem Merge nochmal checken, ob der Practitioner Entry dann nicht doppelt hier drin ist... ich hatte an dieser Stelle auch rumgeschraubt in meinem Mergerequest..

Reply:
ok, schauen wir vor dem merge von deinem request nochmal